libs/libwebsockets/Makefile

```diff
@@ -97,6 +102,18 @@ ifeq ($(BUILD_VARIANT),full)
     CMAKE_OPTIONS += -DLWS_WITH_CGI=ON
 endif
 
+ifeq ($(BUILD_VARIANT),minimal-server)
+   CMAKE_OPTIONS += -DLWS_WITH_STATIC=OFF
+   CMAKE_OPTIONS += -DLWS_WITH_SHARED=ON
```

Comment:
Not yet, I just want to make sure in case these flags change their default value in newer versions of libwebsockets.